tests/test_modeling_flax_common.py

```diff
@@ -274,7 +281,8 @@ def test_equivalence_flax_to_pt(self):
 
                 # Output all for aggressive testing
                 config.output_hidden_states = True
-                # Pure convolutional models have no attention
+                if self.has_attentions:
```

Comment:
Don't like this too much here either. Can't we check if there is a `output_attentions` in the signature of the forward function and if that's the case then we set `config.output_attentions=True`? This way we have 1 dependency less